apps/desktop/src/renderer/i18n/locales/zh.json

```
@@ -2707,6 +2707,12 @@
     "galleryView": "画廊视图",
     "listView": "列表视图",
     "galleryColumnsLabel": "Skill 卡片列数",
+    "tagFilterTooltip": "按标签筛选",
+    "tagFilterTooltipActive": "按标签筛选（已选 {{count}} 个）",
+    "tagFilterSearch": "搜索标签",
+    "noTagMatches": "没有匹配的标签",
+    "removeTag": "移除 \"{{tag}}\"",
```

Biome (2.5.8)

[error] The key removeTag was already declared.

(lint/suspicious/noDuplicateObjectKeys)